public/src/components/common/welcome_banner0.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Is there a reason there are two of these?

Reply:
I'm not sure how that happened, I'll remove the second one and push that change up.